artifactory/commands/transferinstall/transferinstall.go

```diff
@@ -0,0 +1,36 @@
+package transferinstall
+
+import (
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+)
+
+const (
+	groovyFileName  = "dataTransfer.groovy"
+	jarFileName     = "data-transfer.jar"
+	dataTransferUrl = "https://releases.jfrog.io/artifactory/jfrog-releases/data-transfer"
+	libDir          = "lib"
+)
+
+type InstallTransferCommand struct {
```

Comment:
I changed the name to 'InstallDataTransferPluginCommand'

InstallPluginCommand was implemented as a base service to download/copy files into Artifactory 'plugins' directory and request a reload after the files transfer, it can be used to download any plugin.
InstallDataTransferPluginCommand provide the specific information: the files of the plugin, the download url and the name. it will make it easier to implement installation of future plugins. that was my intention for separating them.